data/pilots/galactic-republic/clone-z-95-headhunter.json

```diff
@@ -157,10 +157,11 @@
       ],
       "standard": true,
       "extended": true,
-      "epic": true
+      "epic": true,
+      "keywords": ["Clone"]
     },
     {
-      "name": "“Hawk”",
+      "name": "“Hawk",
```

Comment:
```suggestion
      "name": "“Hawk“",
```

Reply:
Oh my god, you're updating this to match the points doc :joy: :sob:

Reply:
@doweaver lol, i actually noticed the quotes are horrifically inconsistent all over the place, and IIRC this was the style chosen. I think Hawk was just never fixed, so grabbed it along the way